authentication/config.go

```diff
@@ -25,7 +25,7 @@ type Config struct {
 	authMethod authMethod
 }
 
-type MultiOAuth struct {
+type AuthConfig struct {
```

Comment:
Nitpick - should't this be called `OAuthConfig`?

Reply:
I agree, going to change that now

Reply:
On second thought i think `MultiOAuthConfig` is more appropriate to differentiate it, it handles multiple oauth config types

Reply:
👍 I'll switch this to `OAuthConfig` - in renaming this I was hoping to avoid reusing the term `Multi` here not to overload it, given that's also being used for `Multi Tenant Auth`